solidity/contracts/BTCUtils.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.10;
+pragma solidity 0.8.4;
```

Comment:
Can we set it to `^0.8.4;` to improve the experience for projects using higher compiler versions? FWIW, I think we will bump up the version in `tbtc-v2` repo.